### PR TITLE
Plumbline: HybridSL updates 202411

### LINKED ISSUE #27620: Placeholder: Dynamics SL migration tools
### Describe the issue

Issue to process PRs for the Dynamics SL migration tool

### PR DESCRIPTION
**Pull request by blauerPCL**
#### Summary 

- Hide "Set All Dimensions" action and "Open Purchase Orders" option on SL Company Migration Configuration page.
- Create Accounting Periods
- Create Tax Areas
- Add Permission Sets for objects

#### Work Item(s) 
Fixes #27620

Fixes [AB#559550](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/559550)

### CODE REVIEW

**Comment by blauerPCL** [comment]
@nikolakukrika Here is my latest PR for the Dynamics SL Migration extension.

**Review by nikolakukrika — DISMISSED** [review]

**Comment by JesperSchulz** [comment]
@nikolakukrika, I will create an internal PR. Will ping when done 😊

File: Apps/W1/HybridSL/app/src/codeunits/SLFiscalPeriods.Codeunit.al [hunk 1]
```diff
@@ -0,0 +1,118 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.DataMigration.SL;
+
+using Microsoft.DataMigration;
+using Microsoft.Foundation.Period;
```
Comment (JesperSchulz) [inline Apps/W1/HybridSL/app/src/codeunits/SLFiscalPeriods.Codeunit.al:9]:
```suggestion
namespace Microsoft.DataMigration.SL;

using Microsoft.Foundation.Period;
```

**Comment by JesperSchulz** [comment]
Internal PR processed. Not sure why this build doesn't pass. @aholstrup1, could you assist? This PR passed fine internally.

**Comment by JesperSchulz** [comment]
@aholstrup1 will force merge and update the repo to get this in.

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 27626.

**Pull request merged**